src/util.js

```diff
@@ -100,6 +103,26 @@ export function getInfo(context) {
 ///////////////////////////////////
 
 /**
+* Checks if a lesson with a given path has a README-file.
+* Accepts README-files on the form /README or /README_(ISO_CODE).
+**/
+const getReadMePath = (readmeContext, language, path) => {
+  const readmeContextKeys = readmeContext.keys();
+  const readmePathAndLanguageCode = path + '/README_' + language;
+  const readmePathNoLanguageCode = path + '/README';
+
+  if(readmeContextKeys.indexOf('.' + readmePathAndLanguageCode + '.md') !== -1){
+    return readmePathAndLanguageCode;
+  }
+  else if(readmeContextKeys.indexOf('.' + readmePathNoLanguageCode + '.md') !== -1){
+    if(language === readmeContext('.' + readmePathNoLanguageCode + '.md').frontmatter.language){
+      return path + '/README';
+    }
+  }
+  return -1;
```

Comment:
Hvorfor ikke bare returnere en tom streng her? Så trenger du ingen hasReadme-sjekk lenger oppe, bare sende readmePath direkte ut (for dersom den ikke har readmePath, så sender den jo uansett en tom streng).